viziphant/rasterplot.py

```diff
@@ -605,6 +614,31 @@ def sort_func(x):
     return ax, axhistx, axhisty
 
 
+def plot_raster(spiketrain_list):
+    """
+    This function generates a simple dot plot using 'plot_raster_rates' as the
+    base function. The two histograms from the aforementioned function are
+    hidden and only the raster plot is returned.
+
+    Parameters
+    ----------
+    spiketrain_list : list of neo.SpikeTrain or list of list of neo.SpikeTrain
+        List can either contain Neo SpikeTrains object or lists of Neo
+        SpikeTrains objects.
+
+    Returns
+    -------
+    ax : matplotlib.Axes.axes
+        The handle for the raster plot.
+
+    """
+    ax, axhistx, axhisty = plot_raster_rates(spiketrain_list)
+    axhistx.set_visible(False)
+    axhisty.set_visible(False)
```

Comment:
this is called sweeping under the carpet
this function should have simple and clear _implementation_
I agree to leave it for now like it is currently, we'll rethink it in viziphant-new

Reply:
Point taken. This was done as a compromise.